techmatters/flex-plugins
# Chi-2357 add link contact case

Primary reviewer:

## Description

- This PR adds the blue banner to link from contact to case as well as an Undo button on the orange banner after a case has been removed from the contact.

### Checklist

### Related Issues
Fixes [CHI-2357](https://tech-matters.atlassian.net/browse/CHI-2357)

### Verification steps


- Search for contact
- Click on a contact that has been connected to a case
- Verify that you can see the blue banner when you preview the contact
- Click the "Remove from Case" button to remove the case from the contact
- Verify that there is an "Undo" button on the orange banner
- Click on the "Undo" button to undo the previous action

[CHI-2357]: https://tech-matters.atlassian.net/browse/CHI-2357?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
